### PR TITLE
Switch to using @fontsource for Inter & Inconsolata

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/element-hq/element-web/issues/25969

Updates from Inter 3.18 to Inter 4.1 so has some visual changes

Switches Playwright screenshot tests to new headless mode to make the font rendering match reality, also upgrades to Noble to ensure screenshots match between CI & Docker dev.

No point splitting up the PR as the headless mode/noble PR would have to update all the screenshots, as would this fontsource PR, so we'd have to update it twice.

### CODE REVIEW

File: res/themes/light/css/_fonts.pcss [hunk 1]
```diff
@@ -1,149 +1,6 @@
 /* the 'src' links are relative to the bundle.css, which is in a subdirectory.
  */
 
-/* Inter unexpectedly contains various codepoints which collide with emoji, even
-   when variation-16 is applied to request the emoji variant.  From eyeballing
-   the emoji picker, these are: 20e3, 23cf, 24c2, 25a0-25c1, 2665, 2764, 2b06, 2b1c.
-   Therefore we define a unicode-range to load which excludes the glyphs
-   (to avoid having to maintain a fork of Inter). */
```
Comment (dbkr) [inline res/themes/light/css/_fonts.pcss:8]: Is this not necessary anymore?

**Review by dbkr — APPROVED** [review]
otherwise looks good, I'm assuming all the screenshots are reasonable - that's a lot of 'Load Diff' links to click.

File: res/themes/light/css/_fonts.pcss (hunk 1 above)
Comment (t3chguy) [reply res/themes/light/css/_fonts.pcss:8]: Indeed, not possible but also not necessary. fontsource adds the correct unicode ranges for each font-face definition

Comment (dbkr) [reply res/themes/light/css/_fonts.pcss:8]: Bonus!

**Review by gaelledel — APPROVED** [review]

**Pull request merged**